src/Kliva/Views/MainPage.xaml

```diff
@@ -122,11 +122,11 @@
             <Button x:Name="FilterButton"
                     Style="{StaticResource EmptyButtonStyle}"
                     Foreground="{StaticResource KlivaMainBrush}"
-                    Content="{Binding FilterText}"
-                    FontFamily="{StaticResource OpenSansFontLight}"
                     HorizontalAlignment="Stretch"
                     HorizontalContentAlignment="Left"
-                    Margin="10,5,0,10"
+                    Margin="10,0,0,0" Height="48"
+                    VerticalContentAlignment="Bottom"
+                    Padding="0 0 0 5"
```

Comment:
Thx voor de feedback!
Net gecheckt en kan je zéker geen ongelijk geven: op mobile is het niet goed.
Ga eens proberen om de height enkel te zetten in Desktop mode (want daar vind ik het wel geslaagd).
Ik probeer effe wat, look out for a new pull request! ;-)